doc/enterprise/s3gateway.md

```diff
@@ -204,7 +204,7 @@ The Pachyderm S3 gateway supports the following operations:
 * Get objects: Gets file contents on the HEAD of a branch.
 
 You can use any S3 compatible tool, such as MinIO, AWS CLI, or
-`S3cmd` to interact with the Pachyderm S3 gateway.
```

Comment:
s3cmd should be all lowercase

Reply:
Actually, I think it is S3cmd, but also `s3cmd` according to these:

- https://s3tools.org/s3cmd
- https://github.com/s3tools/s3cmd